contracts/interfaces/IStrategy.sol

```diff
@@ -3,6 +3,13 @@ pragma solidity ^0.8.0;
 
 /// @title IStrategy Interface
 /// @author imToken Labs
+/// @notice Interface for a strategy contract that executes a specific trading strategy.
```

Comment:
->
`@notice Interface for contract that implements a specific trading strategy.`